### PR TITLE
Fixing 'make verify' issues

### PR DESCRIPTION
**Pull request by duglin**
make it so that lint/verify failures actually stop the build.
And then fix any issues found

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by arschles** [comment]
@duglin CI failed

**Comment by MHBauer** [comment]
Since this target is multiple lines, can we shove it in a script and have make call the script?

File: pkg/client/informers/internalinterfaces/internal_interfaces.go [hunk 1]
```diff
@@ -26,7 +26,13 @@ import (
 	time "time"
 )
 
+// NewInternalInformerFunc defines a function that takes a client interface
```
Comment (MHBauer) [inline pkg/client/informers/internalinterfaces/internal_interfaces.go]: Changes to this file will be wiped out when generating code.

**Comment by duglin** [comment]
Grrrrrr checking in generated code! 

-Doug

**Comment by duglin** [comment]
ok CI fixed

File: Makefile [hunk 2]
```diff
@@ -231,10 +231,11 @@ verify: .init .generate_files
 	@$(DOCKER_CMD) sh -c \
 	  'for i in $$(find $(TOP_SRC_DIRS) -name *.go \
 	    | grep -v generated \
+	    | grep -v internal_interfaces.go \
```
Comment (MHBauer) [inline Makefile]: is there an way to just not run it on anything in pkg/client?

Comment (duglin) [inline Makefile]: sure, if that's the rule.

Comment (MHBauer) [inline Makefile]: rule?

everything in pkg/client is generated. There's no point in checking any of it.

Comment (duglin) [inline Makefile]: done

File: pkg/apis/servicecatalog/v1alpha1/doc.go [hunk 3]
```diff
@@ -19,5 +19,7 @@ limitations under the License.
 // +k8s:openapi-gen=true
 // +k8s:defaulter-gen=TypeMeta
 
+// Package v1alpha1 defines all of the versioned (v1alpha1) definitions
+// of the service catalog model.
```
Comment (arschles) [inline pkg/apis/servicecatalog/v1alpha1/doc.go]: does code still get generated with the proper group name (`servicecatalog.k8s.io`) after this comment? same question for the above

Comment (duglin) [inline pkg/apis/servicecatalog/v1alpha1/doc.go]: hmmm, I'm not sure. Wouldn't v1alpha1 still appear in the API version string?  @pmorie ??

Comment (arschles) [inline pkg/apis/servicecatalog/v1alpha1/doc.go]: @duglin can you generate the code and see what group name comes out?

Comment (pmorie) [inline pkg/apis/servicecatalog/v1alpha1/doc.go]: We found some extreme brittleness in this file around adding doc comments - do regenerate the code and ensure that nothing has changed.

Also, we need a 'verify' script for the generated code.

Comment (duglin) [inline pkg/apis/servicecatalog/v1alpha1/doc.go]: from my diffs, I don't see any difference in the generated code with this new comment

**Comment by pmorie** [comment]
LGTM

**Pull request merged**